lib/feed/bloc/feed_bloc.dart

```diff
@@ -120,7 +121,7 @@ class FeedBloc extends Bloc<FeedEvent, FeedState> {
 
   /// Handles clearing any messages from the state
   Future<void> _onFeedClearMessage(FeedClearMessageEvent event, Emitter<FeedState> emit) async {
-    emit(state.copyWith(status: FeedStatus.success, message: null));
+    emit(state.copyWith(status: FeedStatus.failure, message: null));
```

Comment:
Just curious, but why was this changed? From what I'm reading, if we get an error message/snackbar that shows up, the feed will become blank because of this line:

```dart
if (state.status == FeedStatus.failure)
SliverToBoxAdapter(
  child: Container(),
),
```

Is this the intended action? Or maybe I missed something here!

(e.g., we fail to fetch more posts from a feed -> which results in throwing a snackbar for the error -> the entire feed will become blank rather than just showing the snackbar and already loaded posts)

Reply:
Good catch. I made this change because, without it, if we fail to initially load a feed (such as in the first screen recording) we will be stuck with the circular progress bar forever, and I wanted to indicate that we tried and failed, and nothing else is loading.

However, you make a great point that if the feed is already populated, this will wipe it.

I will see if I can come up with another way to handle this!

Reply:
Alright, I just ended up adding a new status, since the new code is technically handling an error scenario that wasn't covered before (a failure to load the initial information about the community), which is different than a failure to load more posts, for example. The end result is that the behavior is the same as the first screen recording, while only for this scenario.

Reply:
Looks good now! Thanks for making those changes 😄